### PR TITLE
Update search-services.bicep with latest branding

### PR DESCRIPTION
**Pull request by pamelafox**
Cognitive Search just became AI Search.

### CODE REVIEW

**Review by wbreza — APPROVED** [review]

File: templates/common/infra/bicep/core/search/search-services.bicep [hunk 1]
```diff
@@ -1,4 +1,4 @@
-metadata description = 'Creates an Azure Cognitive Search instance.'
+metadata description = 'Creates an Azure AI Search instance.'
```
Comment (vhvb1989) [inline templates/common/infra/bicep/core/search/search-services.bicep:1]: do we need to update the azd progress reporter?   I think we are using `"Search service"` for `Microsoft.Search/searchServices`...  Is that the same?

**Review by vhvb1989 — APPROVED** [review]

File: templates/common/infra/bicep/core/search/search-services.bicep (hunk 1 above)
Comment (pamelafox) [reply templates/common/infra/bicep/core/search/search-services.bicep:1]: Search service seems fine, its agnostic, but you could be specific and call it AI Search service.

**Pull request merged**